### PR TITLE
Improved Readme

### PR DESCRIPTION
**Pull request by Adrijaned**
Improved the Readme a bit (part of GCI2017)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+that the ground right around fence is severely darkened. This module also misses
+any sort of gate, although it used to have it in past.
+
+![Fences showcase](readme-showcase.png)
+
+# Getting started
+
+If you want to start joking around with fences, you have currently one option:
+
+1) Press ``` ` ``` key to open console
```
Comment (jellysnake) [inline README.md]: `F1` also works

File: README.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four other blocks, with which it is lying side to side.
```
Comment (vampcat) [inline README.md]: "with which it is lying side to side."
This reads weird. Can we rephrase it?
"up to four other neighbouring blocks"

File: README.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+Currently, however, the module is a little bit bugged, which means, that the
```
Comment (vampcat) [inline README.md]: Remove the comma after means.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,23 @@
...
+fences will not get connected correctly the moment they are placed. However,
+placing an arbitrary block right next to a wrongly placed fence and then mining
+it again should fix the issue. Another problem this module currently have, is
```
Comment (vampcat) [inline README.md]: "this module currently have"
Replace the have with has. Also remove the comma.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,23 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four other blocks, with which it is lying side to side.
+Currently, however, the module is a little bit bugged, which means, that the
+fences will not get connected correctly the moment they are placed. However,
+placing an arbitrary block right next to a wrongly placed fence and then mining
```
Comment (jellysnake) [inline README.md]: "Currently however, the module is a little bit bugged. This means that the fences do not connect correctly when they are placed. Forcing an update by placing a breaking a block adjacent to the fences will cause them to align together."

File: README.md [hunk 6]
```diff
@@ -1,4 +1,23 @@
...
+it again should fix the issue. Another problem this module currently have, is
+that the ground right around fence is severely darkened. This module also misses
```
Comment (vampcat) [inline README.md]: The darkening of ground is most probably due to SSAO. Would be good if you mention it.
"is severely darkened if you have SSAO enabled in your graphics settings."

"This module also misses"
Replace `misses` with `lacks`.

File: README.md [hunk 7]
```diff
@@ -1,4 +1,23 @@
...
+any sort of gate, although it used to have it in past.
```
Comment (vampcat) [inline README.md]: Remove all the newlines you have used, and make this entire paragraph one single line. GH handles the line breaks for you, based on text size.

File: README.md (hunk 5 above)
Comment (jellysnake) [reply README.md]: If you are placing multiple comma's in a rather long sentence then it's probably better to break it up. It'll mean you have less "filler" words, things like the "which means" "however" and the like. This with repeating words/phrases makes it flow badly.

File: README.md (hunk 4 above)
Comment (jellysnake) [inline README.md]: "currently has"

File: README.md [hunk 8]
```diff
@@ -1,4 +1,23 @@
...
+
+![Fences showcase](readme-showcase.png)
+
+# Getting started
+
+If you want to start joking around with fences, you have currently one option:
```
Comment (jellysnake) [inline README.md]: "Start joking" is probably not going to translate well to other people.

File: README.md [hunk 9]
```diff
@@ -1,4 +1,18 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four neighbouring blocks. Currently, however, the module is a little bit bugged. This means that the fences do not connect correctly the very moment they are placed. The issue can be fixed by forcing an update for given fence blocks, for instance by placing and mining arbitrary block right next to the fence. Another problem this module currently has is that the ground right around fence is severely darkened if you have SSAO enabled in your graphics settings. This module also lacks any sort of gate, although it used to have it in past.
```
Comment (mjuvekar7) [inline README.md]: Change the second sentence please. It sounds very unprofessional. Something like "This module is still under development, and has a few bugs" would be much better

Comment (mjuvekar7) [reply README.md]: Also this para should be **after** the Getting Started. Never talk about downfalls at the start.

File: README.md [hunk 10]
```diff
@@ -1,4 +1,18 @@
...
+
+![Fences showcase](https://i.imgur.com/5cT16lG.png)
+
+# Getting started
+
+If you want to start playing with fences, you have currently one single option:
```
Comment (mjuvekar7) [inline README.md]: Change this to "To start playing with fences, follow these steps:" or something like that.

**Pull request merged**